_posts/2023-01-success-digital-signage.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Success Story: WPE in digital signage"
+date: 2023-01-06
+tags: [blogpost, success]
+permalink: /blog/2023-success-digital-signage.html
+preview: Digital signage web rendering players have many advantages and are a trend nowadays.
+thumbnail: /assets/img/logo-digital-signage@2x.png
+---
+
+<div class="success-top">
+<img alt="WPE WebKit in digital signage" align="center" src="{{ '/assets/img/logo-digital-signage@2x.png' | url }}">
+<img alt="WPE" align="center" src="{{ '/assets/img/logo-blue.svg' | url }}">
+</div>
+
+Digital signage web rendering players have many advantages and are a trend nowadays. They allow to use HTML5 for composing the UI, provisioning and scheduling new contents to the screens from the cloud is simple and effortless, they provide a robust environment with an automatic crash recovery system, etc. They are also a great choice for digital signage kiosk deployments.
```

Comment:
How about swapping the first sending in line 16 with 
> Web engines have many advantages as the foundation of digital signage solutions.  

It reads more clearly to me, and then the "they" references are referring to web engines, it makes more sense to me.

/cc @meyerweb